File: 编译文件/源码/fib.go
package main

import (
  "fmt"
  //"math"
)

func fib(n uint) uint {
  if n== 0 {
    return 0
  }else if n== 1 {
    return 1
  }else{
    return fib(n-1)+fib(n-2)
  }
}


func main() {
  i := uint(0)
  for(i<20){
    fmt.Println(fib(i))
    i += 1
  }
  fmt.Println(fib(30))
  fmt.Println("Process finished.")
}

File: 编译文件/fib.lua
function fib(n)
  if n<3 then
    return 1
  else
    return fib(n-1)+fib(n-2)
  end
end

io.write(0,"\n")
for n= 1,19 do
  io.write(fib(n),"\n")
end
io.write("\nProcess finished.")